shark_turbine/kernel/ops/wave_ops.py

```diff
@@ -572,6 +572,8 @@ class Read(CustomOp):
 
     @property
     def indexing_dims(self) -> list[IndexSymbol]:
+        if self.mapping is not None:
+            return list(self.mapping.output_shape)
```

Comment:
I don't quite understand why the mapping.output_shape will be different from `self.memory.type.symbolic_shape`? Doesn't the former also represent "output shape"?

Reply:
So you're saying for the case of iGEMM, the `self.memory.type.symbolic_shape` will be (sym.N, sym.C, sym.H, sym.W) as opposed to something like (sym.M, sym.K)?

Reply:
Actually, yeah re-read your reply, that makes sense, thanks :)